### PR TITLE
Add Subscription page

### PR DESCRIPTION
**Pull request by DavidePrincipi**
Add a Subscription manual page for https://github.com/NethServer/dev/issues/6815

Preview at https://nethserver--59.org.readthedocs.build/projects/ns8/en/59/subscription.html

### CODE REVIEW

File: subscription.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+   Available only on Rocky Linux based systems
+
+A NethServer 8 cluster can be registered to a web portal to enable
+additional services.
+
+When a cluster has an active subscription, the following services are
+enabled:
+
+- Remote support by Nethesis
+- Resources monitoring and web portal link
```
Comment (gsanchietti) [inline subscription.rst:20]:
```suggestion
- Resources monitoring and alerting
```

File: subscription.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- Upload of leader node inventory
+- Automatic updates of core and applications
+- Upload of cluster backup (Enterprise only)
+
+
+Register the cluster
+====================
+
+Depending on your subscription type, log in to https://my.nethserver.com
+or https://my.nethesis.it and obtain a unique subscription token for the
```
Comment (gsanchietti) [inline subscription.rst:30]: Add the link to the portals

File: subscription.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+cluster. Follow the portal-documented procedures to obtain it.
+
+.. warning::
+
+  The subscription token is a secret: never communicate or show it to
+  someone else
```
Comment (gsanchietti) [inline subscription.rst:36]:
```suggestion
  The subscription token is a secret: never communicate or share it with
  someone else
```

File: subscription.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  are granted to the support team. Support connections are routed in a
+  private VPN tunnel.
+
+  When the access is granted, a unique ``Session ID`` secret is displayed:
+  copy and paste it in your support request.
+
+- To end the support session, close the VPN tunnel and revoke any granted
+  access, click :guilabel:`End session`.
+
+The support session is valid only on to reach the leader node. The support
```
Comment (gsanchietti) [inline subscription.rst:85]:
```suggestion
The support session is valid only to reach the leader node. The support
```

File: subscription.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+In the ``Subscription`` page, click the :guilabel:`Remove subscription`
+button. The action must be confirmed.
+
+.. hint::
+
+    The "Remove subscription" procedure within the NethServer interface
+    does not clear the web portal cluster account. To complete the cluster
+    subscription removal and avoid being charged, visit your subscription
+    web portal
```
Comment (gsanchietti) [inline subscription.rst:62]: This should be automatic.

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Review by gsanchietti — APPROVED** [review]

**Review by nrauso — COMMENTED** [review]
ok

**Pull request merged**